### PR TITLE
feat: redesign onboarding screens

### LINKED ISSUE #394: Improve the design of the onboarding of the app
cf. figma

### PR DESCRIPTION
**Pull request by justinenerio**
## Changes

- Redesign initial onboarding screens and puzzle reminder

## Related issues

Fixes #394

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [x] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

**Comment by ookami-kb** [comment]
@justinenerio tests are failing. Please also merge with the latest master, I've updated FLutter there.

**Comment by justinenerio** [comment]
@ookami-kb merged with latest master and fixed tests.

File: packages/cryptoplease_ui/lib/src/info_widget.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import 'package:cryptoplease_ui/cryptoplease_ui.dart';
+import 'package:flutter/material.dart';
+
+class CpInfoWidget extends StatelessWidget {
```
Comment (ookami-kb) [inline packages/cryptoplease_ui/lib/src/info_widget.dart:4]: For all the widgets in `cryptoplease_ui` we add a corresponding story in the storybook. Please, add the story for this widget.

**Review by ookami-kb — APPROVED** [review]
@justinenerio LGTM, just add a new story from my comment as a separate PR please.

**Pull request merged**